docs/kit/hardware/kch.md

```diff
@@ -0,0 +1,86 @@
+# KCH (Raspberry Pi HAT)
+
+![3D Render of KCH](img/kch-3d-render.png)
+
+## KCH LEDs
+
+The LEDs on the KCH are split into several groups, some indicate the power state of the KCH, some indicate the state of the robot and some are directly controlled by student code.
+
+### Power Supply
+
+- `5V` (Green)
+    - 5V power is present, whether from the USB-C port on the Pi or the KCH regulator.
+- `Reg` (Green)
+    - The regulator on the KCH is enabled.
+- `Power in` (Green)
+    - 12V power is present on the 12V input.
+- `Reverse polarity` (Red)
+    - The 12V power input has reversed polarity.
+
+### Boot Progress LEDs
+
+There are 5 green LEDs that are a "progress bar" of boot progress on the KCH. Each represents a different stage of boot:
+
+- 20%
+    - Connected to GPIO 7, which is configured as [SPI0 CE1 (ALT0)](https://pinout.xyz/pinout/pin26_gpio7)
+    - It is configured in the [KCH EEPROM](https://github.com/srobo/kch-hw/blob/main/EEPROM/eeprom_settings.txt#L69).
+    - In normal behaviour, it turns on when the EEPROM device tree overlay is loaded, i.e in the early stages of the kernel.
```

Comment:
Boot 20% should light even with an invalid kernel, as long as start*.elf is present.
```suggestion
    - In normal behaviour, it turns on when the GPIO map is loaded from the EEPROM, i.e in the videocore bootloader (which requires a start*.elf).
```

Reply:
Thanks Will. I had wrongly assumed that the device tree was only applied when a kernel was actually present to execute.

Reply:
This change doesn't look to have been applied.

Reply:
> This change doesn't look to have been applied.

I did it manually rather than using the GitHub UI as I had other manual changes to make too.